### PR TITLE
Fix adapter csproj files

### PR DESCRIPTION
**Pull request by garypretty**
Addresses #2814 and #2813

- Added source link package to all adapter projects
- Updated package versions for Twilio / Slack to preview
- Added appropriate properties to CSPROJ to align with other projects
- Ensure package is generated on build for Facebook / Slack

### CODE REVIEW

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Luis/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.QnA/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.ApplicationInsights/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.TemplateManager/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Configuration/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Connector/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Schema/4.5.3)

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</PackageVersion>
+    <Configurations>Debug;Release</Configurations>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Microsoft.Bot.Builder.Adapters.Facebook</PackageId>
+    <Description>Library for connecting bots with Facebook.</Description>
+    <Summary>This library implements C# classes for facebook adapter</Summary>
```
Comment (cleemullins) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/Microsoft.Bot.Builder.Adapters.Facebook.csproj]: "F" not "f".

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+    <PackageVersion Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</PackageVersion>
+    <Configurations>Debug;Release;</Configurations>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>Microsoft.Bot.Builder.Adapters.Slack</PackageId>
+    <Description>Library for connecting bots with Slack.</Description>
+    <Summary>This library implements C# classes for the Slack Adapter</Summary>
```
Comment (cleemullins) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj]: Need a "." at the end.

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
-    <SignAssembly>true</SignAssembly>
-    <DelaySign>true</DelaySign>
-    <AssemblyOriginatorKeyFile>..\..\..\build\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Adapters.Slack.xml</DocumentationFile>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="11.0.1" />
```
Comment (cleemullins) [inline libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj]: This line makes me nervous, as the rest of our SDK uses v10. Can this be changed? If not, what are the implications?

File: libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj (hunk 2 above)
Comment (cleemullins) [reply libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj]: fixed

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.Luis/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.AI.QnA/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.ApplicationInsights/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Azure/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.Dialogs/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Builder.TemplateManager/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Configuration/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Connector/4.5.3)
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll** compared against [version 4.5.3](https://www.nuget.org/packages/Microsoft.Bot.Schema/4.5.3)

**Pull request merged**